app/src/main/java/com/example/whatsapp_clone/Model/Token.java

```diff
@@ -5,5 +5,5 @@
 
 public class Token {
     @SerializedName("authorization")
-    String token;
+    public String token;
```

Comment:
why is it public? will it not be better if we use getter and setter?

Reply:
It is just a class that holds a string there is no need to abstract or encapsulate any logic therefore the best access modifier is public